src/naming.tex

```diff
@@ -95,33 +95,40 @@ \section{Additional Standard Extension Names}
 
 \section{Supervisor-level Instruction-Set Extensions}
 
-Standard supervisor-level instruction-set extensions are defined in Volume II,
-but are named using ``S'' as a prefix, followed by an alphabetical name and an
-optional version number.  Supervisor-level extensions must be separated from
-other multi-letter extensions by an underscore.
+Standard extensions that extend the supervisor-level virtual-memory
+architecture are prefixed with the letters ``Sv'', followed by an alphabetical
+name and an optional version number.  Other standard extensions that extend
+the supervisor-level architecture are prefixed with the letters ``Ss'',
+followed by an alphabetical name and an optional version number.  Such
+extensions are defined in Volume II.
 
 Standard supervisor-level extensions should be listed after standard
 unprivileged extensions.  If multiple supervisor-level extensions are listed,
-they should be ordered alphabetically.
+they should be ordered alphabetically.  Supervisor-level extensions must be
+separated from other multi-letter extensions by an underscore.
 
-\section{Hypervisor-level Instruction-Set Extensions}
+\section{Hypervisor Instruction-Set Extensions}
 
-Standard hypervisor-level instruction-set extensions are named like
-supervisor-level extensions, but beginning with the letter ``H'' instead of
-the letter ``S''.
+Standard hypervisor instruction-set extensions are prefixed with the letter
+``H'', followed by an alphabetical name and an optional version number.
+Such extensions are defined in Volume II.
 
-Standard hypervisor-level extensions should be listed after standard
-lesser-privileged extensions.  If multiple hypervisor-level extensions are
-listed, they should be ordered alphabetically.
+Standard hypervisor extensions should be listed after standard unprivileged
+and supervisor-level extensions.  If multiple hypervisor extensions are
+listed, they should be ordered alphabetically.  Hypervisor extensions
+must be separated from other multi-letter extensions by an underscore.
 
 \section{Machine-level Instruction-Set Extensions}
 
-Standard machine-level instruction-set extensions are prefixed with the three
-letters ``Zxm''.
+Standard extensions that only extend the machine-level architecture are
```

Comment:
Can you please elaborate what scenario is classified as "only extend the machine-level architecture"?

For example, if an extension adds both machine-level CSRs and supervisor-level CSRs, does it fall into the category?
If not, so [https://github.com/riscv/riscv-state-enable/](Smstateen) should really be named as **Ssstateen**?